services/catalog/src/simcore_service_catalog/cli.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+import typer
+from settings_library.utils_cli import create_settings_command
+
+from ._meta import PROJECT_NAME
+from .core.settings import ApplicationSettings
+
+log = logging.getLogger(__name__)
+
+# NOTE: 'main' variable is referred in the setup's entrypoint!
+main = typer.Typer(name=PROJECT_NAME)
+
+main.command()(create_settings_command(settings_cls=ApplicationSettings, logger=log))
+
+
+@main.command()
+def run():
+    """Runs application"""
+    typer.secho("Sorry, this entrypoint is intentionally disabled. Use instead")
+    typer.secho(
+        "$ uvicorn simcore_service_catalog.main:the_app",
+        fg=typer.colors.BLUE,
+    )
```

Comment:
Minor: just thinking: maybe this should return a non 0 exit code. Since it looks and feels like an error when reading the message

Reply:
this is done automatically by click (under typer) when the function does not raise